requirements/docs.txt

```diff
@@ -107,29 +107,29 @@ nbformat==5.10.4
     #   nbclient
     #   nbconvert
     #   nbsphinx
-nbsphinx==0.9.4
+nbsphinx==0.9.5
     # via -r docs.in
 pandocfilters==1.5.1
     # via nbconvert
 pillow==10.4.0
     # via
     #   ipympl
     #   matplotlib
-plopp==24.6.0
+plopp==24.9.0
```

Comment:
3d plots are broken. See https://github.com/scipp/scippneutron/pull/543#issuecomment-2349060835 for context and @nvaytet's comment for a solution.

E.g., in `user-guide/coordinate-transformations.html`